libopenage/renderer/demo/util.h

```diff
@@ -1,7 +1,9 @@
-// Copyright 2023-2023 the openage authors. See copying.md for legal info.
+// Copyright 2023-2024 the openage authors. See copying.md for legal info.
 
 #pragma once
 
+#include <vector>
+#include "renderer/renderable.h"
```

Comment:
Small nitpick but you should add a space between external imports from the standard library and internal import from openage.

```suggestion
#include <vector>

#include "renderer/renderable.h"
```